Gopkg.toml

```diff
@@ -57,7 +60,7 @@
   name = "github.com/gravitational/roundtrip"
 
 [[constraint]]
-  version = "1.1.7"
+  version = "1.1.8"
```

Comment:
I think it's safe and recommended to use the latest release (1.1.10) here (and in gravity) as it fixes a number of issues.